src/vs/workbench/electron-browser/actions/developerActions.ts

```diff
@@ -193,19 +196,24 @@ export class ToggleScreencastModeAction extends Action {
 			const label = keybinding.getLabel();
 
 			if (!event.ctrlKey && !event.altKey && !event.metaKey && !event.shiftKey && this.keybindingService.mightProducePrintableCharacter(event) && label) {
-				keyboardMarker.textContent += ' ' + label;
+				if (previousKeyCode !== event.keyCode || !(event.keyCode === KeyCode.Backspace || event.keyCode === KeyCode.Escape)) {
```

Comment:
I think it would be enough to just add

```typescript
&& !(event.keyCode === KeyCode.Backspace || event.keyCode === KeyCode.Escape)
```

to the if at line 198 and then we can drop the logic built around `previousKeyCode`.

Reply:
This does slightly change the behavior, but I think this would be better.